### ISSUE TITLE
Cannot install k3s rootless on Debian

### ISSUE THREAD

**Issue by frgomes** [PyratLabs/ansible-role-k3s#161]
### Summary

I'm trying to install k3s rootless on Debian 11 and I've stumbled on a situation which suggests that requirements applicable to CentOS are being checked against a Debian distribution. 

### Issue Type

- Bug Report

### Steps to Reproduce

Distribution: Debian 11

```yaml
    - name: Install requirements for k3s rootless
      package:
        name: uidmap
        state: present
      become: true

    - include_role:
        name: xanmanning.k3s
      vars:
        ansible_python_interpreter: "{{vm[0].ansible_python_interpreter | default('/usr/bin/python3')}}"
        k3s_become_for_all: true
        k3s_use_experimental: true
        k3s_start_on_boot: false
        k3s_server:
          rootless: true
          data-dir: "/mnt/data00/k3s/"
          default-local-storage-path: "/mnt/data00/storage/"
          disable:
            - traefik
            - metrics-server
```

### Expected Result

I would expect that the requirements would be met on Debian 11.

### Actual Result

The installation complains that requirements have not been met.

A quick investigation on the code pointed out to the condition which I've commented out as shown below. After commenting out this condition, the installation is able to go ahead.

```yaml
- name: Check user namespaces kernel parameters are adequate
  ansible.builtin.assert:
    that:
      - k3s_get_unprivileged_userns_clone['content'] | b64decode | int == 1
      ###XXX - k3s_get_max_user_namespaces['content'] | b64decode | int >= 28633
      - k3s_current_user_subuid != "UserNotFound:0:0"
      - k3s_current_user_subgid != "UserNotFound:0:0"
      - k3s_current_user_subuid.split(':')[2] | int >= 65536
      - k3s_current_user_subgid.split(':')[2] | int >= 65536
      - ansible_env['XDG_RUNTIME_DIR'] is defined
      - k3s_check_newuidmap_installed.rc == 0
    success_msg: All kernel parameters passed
    fail_msg: >-
      Kernel parameters are not set correctly, please check
      https://github.com/rootless-containers/rootlesskit
```

I've found [this answer](https://stackoverflow.com/a/64448909/62131) on StackOverflow which condenses pretty well various distributions and their corresponding requirements. It's clearly based on the documentation on rootless containers, but it is put together in a way which is visually easier to understand.

**Comment by xanmanning** [comment]
Should be fixed in `v2.12.0`

**Comment by crutonjohn** [comment]
@frgomes is this resolved?
